libs/ui-toolkit/src/components/splash/splash.tsx

```diff
@@ -10,5 +10,5 @@ export const Splash = ({ children }: SplashProps) => {
     'w-full h-full',
     'flex items-center justify-center'
   );
-  return <div className={splashClasses}>{children}</div>;
+  return <span className={splashClasses}>{children}</span>;
```

Comment:
Question (blocking?): Have you checked that this doesn't break other areas where it's used? This could be a big visual change

Reply:
yeah, checked most of the places where this was used. 
I was trying to use the async renderer and this div was causing me problems. 
now that I am not using async renderer anymore this can be reverted. I'll update